io_scene_xray/properties/__init__.py

```diff
@@ -0,0 +1,27 @@
+
+import bpy
+
+
+def gen_flag_prop(mask, description='', customprop=''):
+    def getter(self):
+        return self.flags & mask
+
+    def setter(self, value):
+        self.flags = self.flags | mask if value else self.flags & ~mask
+        if customprop and hasattr(self, customprop):
+            setattr(self, customprop, True)
+
+    return bpy.props.BoolProperty(
+        description=description,
+        get=getter, set=setter,
+        options={'SKIP_SAVE'}
+    )
+
+
+from . import object_
+from . import mesh
+from . import material
+from . import armature
+from . import bone
+from . import action
+from . import scene
```

Comment:
I suggest running `pylint` against changed files, e.g. `pylint io_scene_xray/properties/__init__.py`. This could catch issues like `Imports are always put at the top of the file` (by [PEP8](https://www.python.org/dev/peps/pep-0008/#imports)).

Reply:
Здесь будет ошибка, если импорт раньше функции gen_flag_prop. Так как некоторые импортируемые модули используют эту функцию. И она должна быть объявлена раньше импорта.

Reply:
Хм, получается циклическая зависимость, если я правильно понял. Если так, то давай лучше вынесем ту функцию в отдельный файл. Ибо ну нафиг такие сложности, когда у тебя часть модуля доступна для импорта в другие модули, а часть нет.

Reply:
Про циклическую зависимость я не подумал. Если переносить в отдельный модуль, то как его назвать и куда его поместить? Нужно, чтобы этот модуль был общим для object_ пакета модулей.

Reply:
Ну, мы обычно непонятные модули называем либо `utils`, либо `common`)